lib/parser.js

```diff
@@ -1334,9 +1336,12 @@ replacements['windoge']='window'
 
 module.exports = function parse (line) {
 
-    var tokens = line.match(/'[^']+'|\S+/g);
+    var tokens = tokenizer.tokenize(line);
 
-    if (tokens === null) return line + '\n';
+    if (tokens === null || tokens.length == 0)
```

Comment:
Does tokenize ever return null?  What would that mean?

Reply:
It shouldn’t. I can fix this. The regex used to return a no match but since we control what’s returned then the empty array should do just fine.